ui/packages/consul-ui/ember-cli-build.js

```diff
@@ -147,20 +179,31 @@ module.exports = function(defaults, $ = process.env) {
       },
     }
   );
+  const build = function(path, options) {
+    const {root, ...rest} = options;
+    if(exists(`${root}/${path}`)) {
+      app.import(path, rest);
+    }
+  };
   apps.forEach(item => {
-    app.import(`vendor/${item.name}/routes.js`, {
+    build(`vendor/${item.name}/routes.js`, {
+      root: item.path,
       outputFile: `assets/${item.name}/routes.js`,
     });
-  });
-  [
-    'consul-ui/services'
-    ].concat(devlike ? [
-      'consul-ui/services-debug',
-      'consul-ui/routes-debug'
-    ] : []).forEach(item => {
-      app.import(`vendor/${item}.js`, {
-        outputFile: `assets/${item}.js`,
+    build(`vendor/${item.name}/services.js`, {
+      root: item.path,
+      outputFile: `assets/${item.name}/services.js`,
+    });
+    if(devlike) {
+      build(`vendor/${item.name}/routes-debug.js`, {
+        root: item.path,
+        outputFile: `assets/${item.name}/routes-debug.js`,
       });
+      build(`vendor/${item.name}/services-debug.js`, {
+        root: item.path,
+        outputFile: `assets/${item.name}/services-debug.js`,
+      });
+    }
```

Comment:
Previously we would blindly run `app.import` without checking for file existance. The new `build` function just 'decorates' `app.import` with a file existence check.